### PR TITLE
Check move to pose

### PR DESCRIPTION
**Pull request by hillelv**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/commands/auto/MoveToPose.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import edu.greenblitz.robotName.subsystems.swerve.Chassis.ChassisConstants;
+import edu.greenblitz.robotName.subsystems.swerve.Chassis.SwerveChassis;
+import edu.greenblitz.robotName.utils.FMSUtils;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class MoveToPose {
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/auto/MoveToPose.java:17]: is this a command? if not dont put this in command folder

File: src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java [hunk 2]
```diff
@@ -8,6 +14,8 @@
 
 public class ChassisConstants {
 
+    public static final double ROBOT_WIDTH_IN_METERS = 0.79;
+    public static final double BUMPER_WIDTH = 0.13;
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java:18]: no need to split them

File: src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java [hunk 3]
```diff
@@ -37,6 +46,30 @@ public class ChassisConstants {
...
+    public static final PIDConstants ROTATION_PID = new PIDConstants(3.6, 0, 0);
+    public static final double TOTAL_ERROR_FOR_REPLANNING = 0.5;
+    public static final double ERROR_SPIKE_FOR_REPLANNING = 1;
+    public static final HolonomicPathFollowerConfig PATH_FOLLOWER_CONFIG = new HolonomicPathFollowerConfig(
+            TRANSLATION_PID,
+            ROTATION_PID,
+            MAX_VELOCITY,
+            Math.sqrt(
+                    Math.pow((ROBOT_WIDTH_IN_METERS+BUMPER_WIDTH),2) * 2
+            ),
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java:59]: just save a radius in robot constants

File: src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/SwerveModuleFactory.java [hunk 4]
```diff
@@ -14,7 +14,7 @@ public static ISwerveModule create(SwerveChassis.Module module) {
 
         return switch (RobotConstants.ROBOT_TYPE) {
             case REPLAY -> new ReplaySwerveModule();
-            case ROBOT_NAME -> new MK4ISwerveModule(module);
+            case ROBOT_NAME -> new KazaSwerveModule(module);
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/SwerveModuleFactory.java:17]: give the sds back

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]
good job

**Review by NoamRosenberg08 — CHANGES_REQUESTED** [review]

**Review by YoniKiriaty — APPROVED** [review]
Amireeeee - toda lakaba

File: src/main/java/edu/greenblitz/robotName/Robot.java [hunk 5]
```diff
@@ -50,6 +60,17 @@ public void robotPeriodic() {
...
     }
 
+    private void initializeAutonomousBuilder(){
+        AutoBuilder.configureHolonomic(
+                SwerveChassis.getInstance()::getRobotPose,
+                SwerveChassis.getInstance()::resetChassisPose,
+                SwerveChassis.getInstance()::getRobotRelativeChassisSpeeds,
+                SwerveChassis.getInstance()::moveByRobotRelativeSpeeds,
+                ChassisConstants.PATH_FOLLOWER_CONFIG,
+                () -> FMSUtils.getAlliance() == DriverStation.Alliance.Red,
```
Comment (Stormageddon37) [inline src/main/java/edu/greenblitz/robotName/Robot.java:70]: Why is this specific to the red alliance?

@NoamRosenberg08 says blue should the default

**Review by Stormageddon37 — CHANGES_REQUESTED** [review]
Pretty good, just check the alliance color and
### **CTRL + ALT + L**

File: src/main/java/edu/greenblitz/robotName/Robot.java (hunk 5 above)
Comment (hillelv) [reply src/main/java/edu/greenblitz/robotName/Robot.java:70]: the boolean is "is flipped" it will be true if our alliance is red

**Review by Stormageddon37 — APPROVED** [review]
Nice job, thanks for not submitting 49 files :)

**Review by aaa-png — APPROVED** [review]

**Pull request merged**